src/app/api/expenses/route.ts

```
@@ -0,0 +1,558 @@
+    }
+  }
+}
+
+function calculateSplitAmounts(
+  amount: number,
+  participants: ParticipantData[],
+  splitMethod: string
+): CalculatedParticipant[] {
+  switch (splitMethod) {
+    case "EQUAL":
+      const equalAmount = amount / participants.length;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)